crates/jxl-image/src/lib.rs

```diff
@@ -503,65 +515,289 @@ impl<Ctx> Bundle<Ctx> for BitDepth {
 #[allow(clippy::excessive_precision)]
 impl ImageMetadata {
     const D_UP2: [f32; 15] = [
-        -0.01716200, -0.03452303, -0.04022174, -0.02921014, -0.00624645,
-        0.14111091, 0.28896755, 0.00278718, -0.01610267, 0.56661550,
-        0.03777607, -0.01986694, -0.03144731, -0.01185068, -0.00213539,
+        -0.01716200,
```

Comment:
@tirr-c, Shall we use rustfmt for the new code?

Reply:
oh i did not want to format that file. That blew up the lines. Gonna turn autoformatting off in my IDE

Reply:
@EugeneVert I'd like to do that, yes. Rustfmt properly handles let-else syntax now, so I guess it's time to do formatting!